test/38-sys-info.spec.ts

```diff
@@ -10,16 +10,34 @@ function toMb(bytes: number) {
 }
 
 describe('System information', async () => {
-	const fsSizeStub = stub(systeminformation, 'fsSize');
-	const memStub = stub(systeminformation, 'mem');
-	const currentLoadStub = stub(systeminformation, 'currentLoad');
-	const cpuTempStub = stub(systeminformation, 'cpuTemperature');
+	let fsSizeStub: SinonStub;
+	let memStub: SinonStub;
+	let currentLoadStub: SinonStub;
+	let cpuTempStub: SinonStub;
+	let fsStub: SinonStub;
+	let undervoltStub: SinonStub;
+
+	before(() => {
+		fsSizeStub = stub(systeminformation, 'fsSize');
+		memStub = stub(systeminformation, 'mem').resolves(mockMemory);
+		currentLoadStub = stub(systeminformation, 'currentLoad').resolves(
+			mockCPU.load,
+		);
+		cpuTempStub = stub(systeminformation, 'cpuTemperature').resolves(
+			mockCPU.temp,
+		);
+		// @ts-ignore TS thinks we can't return a buffer...
+		fsStub = stub(fs, 'readFile').resolves(mockCPU.id);
```

Comment:
I'd prefer if the value was called `mockCPU.idBuffer` so it won't imply that Ids are always a buffer of bytes when really it's only a buffer because it was store in a file.